Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -5545,6 +5564,166 @@ script:
     - contextPath: Panorama.Monitor.Message
       description: The message of the logs query.
       type: String
+    - contextPath: Panorama.Monitor.LogType
+      description: The log type of the query.
+      type: String
+    - contextPath: Panorama.Monitor.Logs
+      description: The logs are only returned when polling is enabled and logs are retrieved successfully.
+      type: Unknown
+    - contextPath: Panorama.Monitor.Logs.Action
+      description: The action taken for the session. Can be "alert", "allow", "deny", "drop", "drop-all-packets", "reset-client", "reset-server", "reset-both", or "block-url".
+      type: String
+    - contextPath: Panorama.Monitor.Logs.Application
+      description: The application associated with the session.
+      type: String
+    - contextPath: Panorama.Monitor.Logs.Category
+      description: The URL category of the URL subtype. For WildFire subtype, it is the verdict on the file, and can be either "malicious", "phishing", "grayware", or "benign". For other subtypes, the value is "any".
+      type: String
+    - contextPath: Panorama.Monitor.Logs.DeviceName
+      description: The hostname of the firewall on which the session was logged.
+      type: String
+    - contextPath: Panorama.Monitor.Logs.DestinationAddress
+      description: The original session destination IP address.
+      type: String
+    - contextPath: Panorama.Monitor.Logs.DestinationUser
+      description: The username of the user to which the session was destined.
+      type: String
```

Comment:
this context output is returned only if `polling=true`?

Reply:
That's correct. I am not sure where to indicate this though.